src/cli/utils/flags.go

```diff
@@ -121,7 +122,22 @@ func AddUserFlag(cmd *cobra.Command) {
 		&UserFV,
 		UserFN, nil,
 		"Backup a specific user's data; accepts '"+Wildcard+"' to select all users.")
-	cobra.CheckErr(cmd.MarkFlagRequired(UserFN))
+
+	// if we are marking the user flag as deprecated,
```

Comment:
@ryanfkeepers please let me know if there is a better way to handle this situation